weaviate/batch/crud_batch.py

```diff
@@ -165,29 +192,40 @@ def __init__(self, connection: Connection):
         self._connection = connection
         self._objects_batch = ObjectsBatchRequest()
         self._reference_batch = ReferenceBatchRequest()
-        self._objects_per_second_frame = deque(
-            maxlen=5
+        # do not keep too many past values, so it is a better estimation of the throughput
+        # the throughput is computed for 1 second
+        self._objects_throughput_frame = deque(
+            maxlen=5,
         )
-        self._references_per_second_frame = deque(
-            maxlen=5
+        self._references_throughput_frame = deque(
+            maxlen=5,
```

Comment:
I think you need to multiply the length with the number of workers, otherwise there will not be any history

Reply:
So the reason I left it to 5 is to no take into account the "old" batches. This was we have the most accurate estimation of Weaviate throughput. I do NOT want to link it to number of threads or number of workers because if someone is setting a low value than the estimation is going to be wrong or influenced ONLY by the last batch. We can set a higher value by default or do something like this: `min(max(10, OS_THREAD_COUNT), 16)` or something similar.

Reply:
Hmm I don't fully understand that.
If there is one worker and the len is 5, then it is affected by the last 5 batches. If there are 10 workers and len 5, then it is affected by half of the last round of batches?

Reply:
Yes, since all of them are running more or less at the same time it means that the slower ones are going to be added last, which is nice because we will not over estimate the batch size.

I am also fine with doing something like this: `max(5, OS_THREAD_COUNT)`, or any value instead of 5 (not too large, otherwise if Weaviate gets slower the estimated value will be diluted by the huge history we keep).

Reply:
oki, then let's do `max(5, OS_THREAD_COUNT)`